freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncService.java

```diff
@@ -67,6 +69,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(UserSyncService.class);
 
+    private static final String INTERNAL_USER_CRN = new InternalCrnBuilder(Crn.Service.IAM).getInternalCrnForServiceAsString();
```

Comment:
why is it `IAM` and not `FREEIPA` as this service is not `IAM`?

Reply:
Thunderhead only recognizes the internal user if the service is IAM.

Reply:
There are other places as well where we use the internal crn to talk to UMS with FREEIPA service type

Reply:
Verified locally and it doesn't work when `freeipa` is used instead of `iam`.